pants

```diff
@@ -245,32 +267,35 @@ function bootstrap_pants {
    fi
   local python_major_minor_version
   python_major_minor_version="$(get_python_major_minor_version "${python}")"
-  local target_folder_name
-  target_folder_name="${pants_version}_py${python_major_minor_version}"
+  local target_folder_name="${pants_version}_py${python_major_minor_version}"
+  local bootstrapped="${PANTS_BOOTSTRAP}/${target_folder_name}"
 
-  if [[ ! -d "${PANTS_BOOTSTRAP}/${target_folder_name}" ]]; then
+  if [[ ! -d "${bootstrapped}" ]]; then
     (
-      local venv_path
-      venv_path="$(bootstrap_venv)"
+      green "Bootstrapping Pants using ${python}"
       local staging_dir
       staging_dir=$(tempdir "${PANTS_BOOTSTRAP}")
+      local virtualenv_path
+      virtualenv_path="$(bootstrap_virtualenv "${python}")" || exit 1
+      green "Installing ${pants_requirement} into a virtual environment at ${bootstrapped}"
       # shellcheck disable=SC2086
-      "${python}" "${venv_path}/virtualenv.py" --no-download "${staging_dir}/install" && \
+      ${python} "${virtualenv_path}" --no-download "${staging_dir}/install" && \
```

Comment:
The quoting of `${python}` was correct. `/Users/Jane Doe/...` as a home for python is both valid and likelyish.